src/main/archetype/dispatcher.cloud/src/conf.dispatcher.d/filters/default_filters.any

```diff
@@ -90,8 +90,8 @@
 /0054 { /type "allow" /method "GET" /path "/content/*" /selectors 'sitemap(-index)?' /extension "xml" }
 
 # Allow GraphQL & preflight requests
-# GraphQL also supports "GET" requests, if you intend to use "GET" add a rule in filters.any
-/0060 { /type "allow" /method '(POST|OPTIONS)' /url "/content/_cq_graphql/*/endpoint.json" }
+# GraphQL also supports "GET" or "POST" requests, if you intend to use "GET" or "POST" add a rule in filters.any
+/0060 { /type "allow" /method '(OPTIONS)' /url "/content/_cq_graphql/*/endpoint.json" }
```

Comment:
I understand we want to block usage of POST on publish.. - why do we still support OPTION? 

Maybe we should - in the comment above - mention that we recommend to use Persisted Queries.

Reply:
@keepthebyte I update the rule and comment accordingly

Reply:
@keepthebyte we have to keep the POST requests available for stage and dev environments so I updated this PR accordingly to allow better filtering for all customers based on environment type see also my comment https://git.corp.adobe.com/Granite/dispatcher-k8s-base/pull/152#issuecomment-4099643